forms/jqm/elements/choiceexpanded.js

```diff
@@ -21,24 +21,35 @@ define(function (require) {
       return ChoiceElementView.prototype.remove.call(this);
     },
 
+    renderLabel: function () {
+      if (!this.$label) {
+        this.$label = $('<legend></legend>');
+      }
+
+      if (!this.$el.find(this.$label).length) {
```

Comment:
You said this last time i used `find`, so I looked it up before and just re-read to confirm i remembered correctly.

Find does not traverse anything when used with two existing selectors: https://github.com/jquery/jquery/blob/master/src/traversing/findFilter.js

Because the argument to find is already a jquery object, the functions are basically doing a [https://developer.mozilla.org/en-US/docs/Web/API/Node/contains](DOMNode.contains)

What I can do is change it from `this.$el.find()` to `this.$fieldset.find()`